src/Netresearch/TimeTrackerBundle/Helper/LdapClient.php

```diff
@@ -223,6 +240,34 @@ public function setBaseDn($base_dn)
 
 
     /**
+     * Determines whether SSL will be used for LDAP-connection or not
+     *
+     * @param boolean $useSSL
+     * @return $this
+     */
+    public function setUseSSL($useSSL)
+    {
+        $this->_useSSL = !empty($useSSL) && $this->_useSSL !== 0;
```

Comment:
i am confused, why so complicated?
What does 0 mean if set for this->_useSSL?
You cannot set useSSL to false!?